client/src/components/Pagination.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import $ from 'jquery';
+
+const Pagination = ({ numReviews, paginate }) => {
+  const pages = [];
+  const numPages = Math.ceil(numReviews / 7);
+  for (let i = 1; i <= numPages; i += 1) {
+    pages.push(i);
+  }
```

Comment:
I suspect you're going to need either a useState hook or to turn this into a class component to have one page formatted at the start.

There may be an easier way, like always initially rendering page 1 that way in the beginning, but I haven't used jQuery to apply CSS like you are. There may be a way to do it with that technique that I'm just not familiar with.

Reply:
Thanks, I'll look into the different options!